StarCellBio/urls.py

```diff
@@ -67,9 +67,10 @@
 
     # Dashboard
     url(r'^ab/assignments/$', instructor_common.assignments, name="common_assignments"),
+    url(r'^ab/assignments/(?P<error>[^/]+)$', instructor_common.assignments, name="common_assignments"),
```

Comment:
What is this for? It points to the same view as the previous line and I don't see any reference to `error` in the view